src/replays.ts

```diff
@@ -233,6 +233,18 @@ export const Replays = new class {
 			SQL`uploadtime, id, format, players, rating`
 		)`WHERE private = 0 ORDER BY uploadtime DESC LIMIT 51`.then(this.toReplays);
 	}
+	
+	getBatch(ids: string[]) {
+		let idsForQuery = '';
+		for (var i = 0; i < ids.length; i++) {
+			const id = ids[i];
+			idsForQuery += `'${id}'`;
+			if (i !== ids.length - 1) idsForQuery += ",";
+		}
+		return replays.selectAll(
+			SQL`*`
+		)`WHERE private = 0 AND id IN (${idsForQuery}) LIMIT 51`.then(this.toReplays);
```

Comment:
```suggestion
		return replays.selectAll()`WHERE private = 0 AND id IN (${idsForQuery}) LIMIT 51`.then(this.toReplays);
```
This is only necessary if you're not selecting `*` - it defaults to `*` otherwise.